refinery/ui/source/js/user-file-browser/ctrls/user-file-browser-files-ctrl.js

```diff
@@ -34,28 +40,27 @@
       userFileSortsService,
       gridOptionsService
   ) {
+    gridOptionsService.appScopeProvider = vm;
+    var promise = $q.defer();
     var vm = this;
-    vm.nodesCount = userFileBrowserFactory.dataSetNodes.nodesCount;
-    vm.totalNodesCount = userFileBrowserFactory.dataSetNodes.totalNodesCount;
     vm.downloadCsv = downloadCsv;
     vm.downloadCsvQuery = downloadCsvQuery;
+    vm.refreshUserFiles = refreshUserFiles;
+    vm.gridOptions = gridOptionsService;
+    vm.nodesCount = userFileBrowserFactory.dataSetNodes.nodesCount;
+    vm.sortChanged = sortChanged;
+    vm.totalNodesCount = userFileBrowserFactory.dataSetNodes.totalNodesCount;
 
-    var promise = $q.defer();
```

Comment:
1) Wrapped service call in a function per style guide and testability.
2) Reordered function per style guide.